src/containers/HolderChange/Review.jsx

```diff
@@ -197,7 +196,18 @@ const Review = (props) => {
           </Grid>
         </Grid>
 
-        <Divider variant="middle" sx={{ mt: '24px', ml: 0, mr: '32px' }} />
+        <Divider
+          variant="middle"
+          sx={{ mt: '24px', ml: 0, mr: '32px', mb: '24px' }}
+        />
+
+        { member === undefined && (
+          <FormHelperText
```

Comment:
I know this code is copied from other fields but it is kind of dangerous to use a third party component, like FormHelperText, outside the context the context it is suposed to be (a Form Field). I guess we choose that component because the typography. I  would make a new component using the Typography of FormHelperText.

Reply:
For coherence, we leave it as it is. We can put it as technical debt if we decide to change those FormHelperText.